guide/resources/design-research.md

```diff
@@ -252,7 +252,7 @@ Documentation of the user research, redesign and testing process of Coinhako, a
 
 #### [Lightning Network UX Research](https://patestevao.com/work/lightning-network-ux-research/)
 
-The study conducted to look into the interactions between users and the lightning payments. It also reviewes three well-known projects at that time and looks through their user-experience.
+The study conducted to look into the interactions between users and the lightning payments. It also reviewers three well-known projects at that time and looks through their user-experience.
```

Comment:
```suggestion
The study conducted to look into the interactions between users and the lightning payments. It also reviews three well-known projects at that time and looks through their user-experience.
```

The proposed change is not quite right, as "reviews" is a verb that refers to the study. So I am counter-proposing an adjustment here.